code/src/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import todoReducer from 'reducers/todoSlice';
+
+export default configureStore({
+  reducer: {
+    todos: todoReducer
+  }
+});
```

Comment:
this could be added directly in the app.js :) and a tip is to add the localstorage for the tasks to save.